db/views/materialized_patient_summaries_v01.sql

```diff
@@ -0,0 +1,118 @@
+SELECT p.recorded_at,
+       CONCAT(date_part('year', p.recorded_at), ' Q', EXTRACT(QUARTER
```

Comment:
This file is intentionally formatted this way so that we can see all the exposed columns properly (this is also valid formatting for SQL, via sqlformat.org).